packages/front-end/pages/settings/index.tsx

```diff
@@ -106,6 +118,18 @@ const GeneralSettingsPage = (): React.ReactElement => {
       const newVal = { ...form.getValues() };
       Object.keys(newVal).forEach((k) => {
         newVal[k] = data.organization.settings?.[k] || newVal[k];
+
+        // Existing values are stored as a multiplier, e.g. 50% on the UI is stored as 0.5
+        // Transform these values from the UI format
+        if (k === "metricDefaults") {
+          newVal.metricDefaults = {
+            ...newVal.metricDefaults,
+            maxPercentageChange:
+              newVal.metricDefaults.maxPercentageChange * 100,
+            minPercentageChange:
+              newVal.metricDefaults.minPercentageChange * 100,
+          };
+        }
```

Comment:
It might be cleaner to keep the form value as a decimal and do the transform in the input instead:

```tsx
<Field
  ...
  value={form.watch("metricDefaults.minPercentChange") * 100}
  onChange={(e) => {
    form.setValue("metricDefaults.minPercentChange", e.target.value / 100
  }}
```

I don't really love either approach to be honest.  We have this exact decimal->percent issue a bunch of places in the UI.  There's got to be a more elegant and reusable way to handle this.

Reply:
I was trying not to change the existing logic out of risk of introducing a bug, but I can take another pass at it if we prefer to clean up this section.

Reply:
As discussed on Slack, we are going to stick with the less-clean version that is currently in the PR. 

The concerning bug where we call `form.reset()` already exists on main. I've created a ticket for us to address the existing bug: https://github.com/growthbook/growthbook/issues/522

In that process, this will heavily change. Not calling `form.reset()` should simplify this form.